api/src/test/java/org/pmiops/workbench/testconfig/fixtures/ReportingUserFixture.java

```diff
@@ -179,7 +180,6 @@ public DbUser createEntity() {
     user.setContactEmail(USER__CONTACT_EMAIL);
     user.setCreationTime(USER__CREATION_TIME);
     user.setCurrentPosition(USER__CURRENT_POSITION);
-    user.setDataAccessLevel(USER__DATA_ACCESS_LEVEL);
```

Comment:
cannot set this value.  Can only derive it from the accessTierShortNames via `AccessTierService.temporaryDataAccessLevelKluge()` 

This is temporary to allow a transition.  It will be removed in RW-6189.